### ISSUE TITLE
Rule proposal: `prefer-positive-condition`

### ISSUE THREAD

**Issue by JounQin** [sindresorhus/eslint-plugin-unicorn#1073]
Positive condition is more intuitive.

## Fail

```js
if (!condition) {
  // do A
} else {
  // do B
}

!condition ? A : B

```

## Pass

```js
if (condition) {
  // do B
} else {
  // do A
}

condition ? B : A
```

**Comment by sindresorhus** [comment]
This already exists in ESLint: https://eslint.org/docs/rules/no-negated-condition
